packages/react-components/src/components/chart/trendCursor/useTrendCursors.ts

```diff
@@ -83,12 +84,13 @@ const useTrendCursors = ({
 
   const trendCursorKeyMap: KeyMap = {
     commandDown: { sequence: 't', action: 'keydown' },
-    commandUp: { sequence: 't', action: 'keyup' },
   };
 
   const trendCursorHandlers = {
-    commandDown: () => setIsInCursorAddMode(true),
-    commandUp: () => setIsInCursorAddMode(false),
+    commandDown: () => {
```

Comment:
I am not sure why commandUp was removed.... if the user continues to hold T, we should keep them in the add mode as long as that ?